install/kubernetes/cilium/templates/cilium-agent/clusterrole.yaml

```diff
@@ -39,7 +39,7 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - nodes
+  - nodes/status
   verbs:
   # To annotate the k8s node with Cilium's metadata
   - update
```

Comment:
wondering if this verb should be patch instead ? 

```
2022-04-27T13:10:34.793533109Z level=warning msg="Unable to patch node resource with annotation" error="nodes \"kind-control-plane\" is forbidden: User \"system:serviceaccount:kube-system:cilium\" cannot patch resource \"nodes/status\" in API group \"\" at the cluster scope" key=0 nodeName=kind-control-plane subsys=k8s v4CiliumHostIP.IPv4=10.0.1.21 v4Prefix=10.0.1.0/24 v4healthIP.IPv4=10.0.1.3 v6CiliumHostIP.IPv6="<nil>" v6Prefix="<nil>" v6healthIP.IPv6="<nil>"
```

```
$ kg clusterrole cilium -o json | jq .rules | grep -A 5 -B 5 nodes/status
  {
    "apiGroups": [
      ""
    ],
    "resources": [
      "nodes/status"
    ],
    "verbs": [
      "update"
    ]
  },

$ ksysg cm cilium-config -o json  | jqd | grep annotate | head -1
    "annotate-k8s-node": "true",

```

Reply:
@sayboras ah it was suppose to be a patch, nice catch!